src/index.html

```diff
@@ -7,6 +7,351 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+            <div class="reactangle"></div>
```

Comment:
why do u need this div?

Reply:
ahah, sorry, i try to do hover reactangle, because didn't read how i can do this